adapter2/src/python.rs

```diff
@@ -10,18 +10,16 @@ pub trait EventSink {
     fn display_html(&self, html: String, title: Option<String>, position: Option<i32>, reveal: bool);
 }
 
-#[allow(unions_with_drop_fields)]
 #[repr(C)]
 union ValueResult {
-    value: SBValue,
-    error: SBError,
+    value: mem::ManuallyDrop<SBValue>,
```

Comment:
I wonder if these should be `MaybeUninit` instead?   `ManuallyDrop` sort of implies that the value is valid (even though it won't be dropped automatically).

Reply:
I went with ```ManuallyDrop``` since these unions are only used internally as a means for getting values from C. All of that is already wrapped in an unsafe block and the return status determines what to get from the union.

You're right that MaybeUninit is in fact a bit clearer, but both options are equally unsafe in their usage. The developer always has to assume, at some point, that the value is valid...